compiler/crates/relay-typegen/src/rescript.rs

```diff
@@ -2878,6 +2880,7 @@ impl Writer for ReScriptPrinter {
                                             ],
                                             instruction: ConverterInstructions::ConvertCustomField(
                                                 scalar_name.clone(),
+                                                found_in_array.clone()
```

Comment:
I added `found_in_array` to `AstToStringNeedsConversion::CustomScalar` just so that I could pass a correct value in `ConverterInstructions::ConvertCustomField`, which fixes another issue I discovered regarding provided values.
Is there an easier way to check if it was in an array here within `write_type_assertion` function?